nengo_gui/examples/tutorial/23-spa-binding.py.cfg

```diff
@@ -1,62 +1,60 @@
+_viz_config[model.color].size=(0.4, 0.21052631578947367)
 _viz_config[model.color].expanded=False
 _viz_config[model.color].has_layout=False
-_viz_config[model.color.state].expanded=False
-_viz_config[model.color.state].has_layout=False
-_viz_config[model.cortical].pos=(0.47179813401187465, 0.3047932253070003)
-_viz_config[model.cortical].size=(0.11764705882352941, 0.11764705882352941)
+_viz_config[model.color.state_ensembles].expanded=False
+_viz_config[model.color.state_ensembles].has_layout=False
+_viz_config[model.cortical].pos=(0.24180837087371448, 0.708807244881999)
+_viz_config[model.cortical].size=(0.1, 0.1)
 _viz_config[model.cortical].expanded=False
-_viz_config[model.cortical].has_layout=True
-_viz_config[model.cortical.bias].pos=(0.5, 0.08333333333333333)
-_viz_config[model.cortical.bias].size=(0.3076923076923077, 0.037037037037037035)
-_viz_config[model.cortical.networks[0]].pos=(0.5, 0.7685185185185184)
-_viz_config[model.cortical.networks[0]].size=(0.4, 0.18518518518518517)
+_viz_config[model.cortical].has_layout=False
 _viz_config[model.cortical.networks[0]].expanded=False
 _viz_config[model.cortical.networks[0]].has_layout=False
 _viz_config[model.cortical.networks[0].product].expanded=False
 _viz_config[model.cortical.networks[0].product].has_layout=False
-_viz_config[model.cortical.networks[0].product.product].expanded=False
-_viz_config[model.cortical.networks[0].product.product].has_layout=False
-_viz_config[model.cortical.spa].pos=(0.2267104891150693, 0.3876331123166566)
-_viz_config[model.cortical.spa].size=(0.7346938775510203, 0.7346938775510203)
+_viz_config[model.cortical.networks[0].product.sq1].expanded=False
+_viz_config[model.cortical.networks[0].product.sq1].has_layout=False
+_viz_config[model.cortical.networks[0].product.sq2].expanded=False
+_viz_config[model.cortical.networks[0].product.sq2].has_layout=False
+_viz_config[model.cortical.spa].pos=(1.1367514145067479, 0.7492759268866362)
+_viz_config[model.cortical.spa].size=(0.4030167999413548, 0.4030167999413548)
 _viz_config[model.cortical.spa].expanded=True
 _viz_config[model.cortical.spa].has_layout=True
-_viz_config[model.memory].pos=(0.8058320444377932, 0.30645085038234693)
-_viz_config[model.memory].size=(0.11764705882352941, 0.11764705882352941)
+_viz_config[model.memory].pos=(0.8961610374063167, 0.7081487830506279)
+_viz_config[model.memory].size=(0.1, 0.1)
 _viz_config[model.memory].expanded=False
 _viz_config[model.memory].has_layout=False
-_viz_config[model.memory.state].expanded=False
-_viz_config[model.memory.state].has_layout=False
-_viz_config[model.shape].pos=(0.08471786321417869, 0.1569041118557128)
-_viz_config[model.shape].size=(0.0968667431223761, 0.07580458343675023)
+_viz_config[model.memory.state_ensembles].expanded=False
+_viz_config[model.memory.state_ensembles].has_layout=False
+_viz_config[model.shape].pos=(0.48323371886886723, 1.2739592850585906)
+_viz_config[model.shape].size=(0.4, 0.21052631578947367)
```

Comment:
Did you mean to reorganize the layout like this?  The spa clouds are now far away from the components they are plotting, which seems strange to me.